test/WebJobs.Script.Tests.Integration/TestScripts/CSharp/QueueTriggerToBlobRichTypes/run.csx

```diff
@@ -1,5 +1,5 @@
-﻿#r "Microsoft.WindowsAzure.Storage"
-using Microsoft.WindowsAzure.Storage.Blob;
+﻿#r "Microsoft.Azure.Storage.Blob"
```

Comment:
Oh maybe my above comment was wrong; I didn't expect this to correctly resolve. @fabiocav does this come from somewhere else?

Reply:
Debugged this a few days ago - its getting resolved by the trusted platform assemblies resolver. @fabiocav thinks that this was a behavior change in Roslyn at some point. This actually means that there are a bunch of other assemblies that can be loaded through #r, such as protobufs and gRPC.

Not sure when this behavior changed exactly, but @fabiocav and I agreed that addressing it is out of scope for this storage update, so I decided to add a test case to guard against accidental regression.